packages/jsii-java-runtime/project/src/main/java/software/amazon/jsii/JsiiObject.java

```diff
@@ -24,14 +20,22 @@
     /**
      * A special constructor that allows creating wrapper objects while bypassing the normal constructor
      * chain. This is used when an object was created in javascript-land and just needs a wrapper in native-land.
-     * @param initializationMode Must always be set to "Jsii".
+     * @param initializationMode Must always be set to "JSII".
      */
     protected JsiiObject(final InitializationMode initializationMode) {
-        if (initializationMode != InitializationMode.Jsii) {
-            throw new JsiiException("The only supported initialization mode is 'Jsii'");
+        if (initializationMode != InitializationMode.JSII) {
+            throw new JsiiException("The only supported initialization mode is 'JSII'");
```

Comment:
Use the enum in the exception?

Reply:
Yeah that's a good idea. Will make that change.